app/js/components/labOrderEntry/LabEntryForm.jsx

```diff
@@ -14,46 +18,54 @@ export class LabEntryForm extends React.Component {
     categoryId: 1,
     defaultTests: [],
     selectedTests: [],
-    selectedPanel: null,
+    selectedPanelIds: [],
     disableSaveButton: true,
     disableCancelButton: true,
   };
 
-  selectTest = (testId) => {
-    if (!this.state.selectedTests.includes(testId) && !this.state.defaultTests.includes(testId)) {
+  selectTest = (test) => {
+    const { addDraftLabOrdersToStore, draftLabOrders, deleteDraftLabOrderFromStore } = this.props;
+    const selectedTestIds = this.state.selectedTests.map(currentTest => currentTest.id);
+    const defaultTestIds = this.state.defaultTests.map(currentTest => currentTest.id);
```

Comment:
You can try composing them up as:
`const isSelectedId = this.state.selectedTests.map(currentTest => currentTest.id).includes(test.id);`

`const isDefaultTestId = this.state.defaultTests.map(currentTest => currentTest.id).includes(test.id);`

Reply:
Done, Thanks